docs/updates/release-notes.md

```diff
@@ -12,6 +12,33 @@ import CloudBadge from '@site/src/components/CloudBadge';
 
 # Latest Release Notes
 
+## 3.15.0 - 2024-04-24 - AAB to APK, Improved Testing Distribution, Publish Event Enhancement
+
+### 🆕 New Features
+
+- [The public link](https://docs.appcircle.io/distribute/create-or-select-a-distribution-profile#using-public-link-for-distribution) in the distribution settings has been converted to a QR code for easier access and sharing. <CloudBadge/>
+- Users can now filter the app version list on [the testing portal](https://docs.appcircle.io/distribute/testing-management/downloading-binaries) by app name, version, release notes, or build number for enhanced navigation and search capabilities. <CloudBadge/>
+- The uploaded or built AAB files are now automatically converted to a universal format. Additionally, the newly created APK file is saved discreetly with the second app's resource id. <CloudBadge/>
+- A new command, ["build active-list"](https://docs.appcircle.io/appcircle-api/) has been added, allowing users to view active builds in the queue directly from their command line interface. <CloudBadge/> <SelfHostedBadge/>
+- A new command, ["build view"](https://docs.appcircle.io/appcircle-api/) has been added, enabling users to access and view detailed information about builds directly from the command line interface. <CloudBadge/> <SelfHostedBadge/>
+- The "Default M1 pool" now includes [Xcode 15.4 beta-1](https://docs.appcircle.io/infrastructure/ios-build-infrastructure#available-xcode-versions) installed on runners. As this is a beta release, we strongly recommend testing your workflows extensively to ensure compatibility and stability. <CloudBadge/> <SelfHostedBadge/>
+
+
+### :muscle: Improvements
+
+- New [Publish Events](https://docs.appcircle.io/publish-module/), such as Publish Profile Created, Publish Profile Deleted, App Version Uploaded, App Version Created, and App Version Deleted, have been introduced to enrich the activity report. <CloudBadge/>
```

Comment:
```suggestion
- We have introduced new [Publish Events](https://docs.appcircle.io/publish-module/) like Publish Profile Created, Publish Profile Deleted, App Version Uploaded, App Version Created, and App Version Deleted to enrich the activity report. <CloudBadge/>
```

Reply:
When creating documentation, please ensure you use active voice wherever possible. This style makes our documentation clearer and more engaging by directly identifying the action and who is performing it. Active voice contributes to more straightforward and effective communication, enhancing the readability and accessibility of our content for all users.